packages/mui-system/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/system",
-  "version": "5.13.2",
+  "version": "5.13.3",
```

Comment:
```suggestion
  "version": "5.13.5",
```

This might be confusing to you but the decision is to use the same version for all stable products.

Since `@mui/monorepo` is `5.13.5`, the system should be `5.13.5` as well.

cc @michaldudak @mj12albert to verify my understanding.

Reply:
Yes, that's correct. It should be 5.13.5.

Reply:
We should document this in the release process.

Reply:
`scripts/README.md` is already updated with missing details.